src/jdk.accessibility/windows/native/include/bridge/AccessBridgePackages.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2015, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
this copyright also needs to be updated..

Reply:
Updated.